recipe/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if [[ `uname` == 'Darwin' ]]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.9
```

Comment:
Yes but that can be fixed using `export CXXFLAGS="${CXXFLAGS} -DBOOST_NO_CXX11_RVALUE_REFERENCES"`.
The problem is I want to make changes to all bob packages and don't know that would be the consequence in the rest of the packages before I merge in the changes.
For example, I want to remove the `-B` option in `python -B setup.y install` but I dont know what will happen until I change `bob.extension` first and then `bob.blitz` and then ...